<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/myappico.png</normaloff>:/myappico.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Imagen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo_base_de_datos"/>
    <addaction name="actionAbrir_Base_de_datos"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevo_base_de_datos"/>
   <addaction name="actionAbrir_Base_de_datos"/>
   <addaction name="separator"/>
   <addaction name="actionInsertar_Persona"/>
   <addaction name="actionEliminar_Persona"/>
   <addaction name="separator"/>
   <addaction name="actionSalir"/>
  </widget>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/salir.png</normaloff>:/salir.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionNuevo_base_de_datos">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/base.png</normaloff>:/base.png</iconset>
   </property>
   <property name="text">
    <string>Nueva Base de datos</string>
   </property>
  </action>
  <action name="actionAbrir_Base_de_datos">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/abrir.png</normaloff>:/abrir.png</iconset>
   </property>
   <property name="text">
    <string>Abrir Base de datos</string>
   </property>
  </action>
  <action name="actionInsertar_Persona">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/agregar.png</normaloff>:/agregar.png</iconset>
   </property>
   <property name="text">
    <string>Insertar Persona</string>
   </property>
  </action>
  <action name="actionEliminar_Persona">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/eliminarPersona.png</normaloff>:/eliminarPersona.png</iconset>
   </property>
   <property name="text">
    <string>Eliminar Persona</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
